packages/MarkdownEditor-Tests.package/MarkdownEmphasisDelimiterTest.class/instance/testDifferentDelimitersCreateBoldAttribute.st

```diff
@@ -1,17 +1,11 @@
-tests - attributes
+tests - emphases
 testDifferentDelimitersCreateBoldAttribute
 
-	| opener closer attribute |
-	opener := MarkdownEmphasisDelimiter new
-					content: '****';
-					startPosition: 1;
-					yourself.
-	closer := MarkdownEmphasisDelimiter new
-					content: '***';
-					startPosition: 6;
-					yourself.
+	| opener closer emphasis |
+	opener := MarkdownEmphasisDelimiter newFor: '****' at: 1.
+	closer := MarkdownEmphasisDelimiter newFor: '***' at: 6.
 					
-	attribute := closer attributeStartingAt: opener. 	
-	self assert: (attribute isKindOf: MarkdownAttributeBold).
-	self assert: 3 equals: attribute startPosition.
-	self assert: 7 equals: attribute endPosition
\ No newline at end of file
+	emphasis := closer attributeStartingAt: opener. 	
+	self assert: TextEmphasis bold equals: emphasis textAttribute.
```

Comment:
You could compare `emphasis` with `MarkdownEmphasis strongFrom: 3 to: 7`.

Applies also to `testEmphasisAttributeItalic`.

Reply:
Kira and me, we discussed this. In my opinion, it is another layer of abstraction because this comparison more feels like integration tests whereas this is a unit test.

Reply:
As discussed with SWA people, both variants are fine. If the comparison gets repeated often (becomes a pattern), a shorter way is preferred (custom assert or just using `=`).
As this happens only a handful of time, I'm indifferent about this